packages/rpc-core/src/rpc-methods/getStakeMinimumDelegation.ts

```diff
@@ -0,0 +1,14 @@
+import { RpcResponse, U64UnsafeBeyond2Pow53Minus1, Commitment } from './common';
+
+type GetStakeMinimumDelegationApiResponse = RpcResponse<U64UnsafeBeyond2Pow53Minus1>;
```

Comment:
👍🏻

Reply:
I think at this point it's worth creating a branded `Lamports` type. See `Base58EncodedAddress` for how to create a branded type.

This will imply that you can create a function that only accepts `Lamports` as a type and if you try to pass it a non-branded `BigInt` you'll get a Typescript error. Should save a couple million in mistake fees.

Reply:
Yeah sure, where do you want it go?

Reply:
I have a `@solana/keys` package which is a nice place to put the address types, but for everything else I’ve just been hucking ‘primitive’ types into global typedefs.

Throw it anywhere for now, just try to avoid circular references between modules.

Reply:
Ok, put it under `common.ts`